### ISSUE TITLE
Angry Pods - Make it possible to configure which pods to be in the game

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/fabric8-console#113]
I would like to avoid killing fluentd / fabric8 web console / etc.

It would be nice to blacklist those core containers out of the box. And also a way to [x] or [ ] which apps to in the game / or RCs etc. But maybe the "app" name we use on overview.

Also maybe chose how big the maze is, so when you have 3 or 4 pods, then you can play in a tiny maze.
And also how much armour yourself has, eg so they have to shoot you much more before you die.

@gashcrumb would be nice to make angry pod more demo friendly as I would to showcase it some more, but what happens is that either the maze is to big and I have to walk to much to find my camel pods. Or they shot me all the time, or I shot a fabric8 web console pod etc.

**Comment by gashcrumb** [comment]
yeah, I can probably tighten up the spawn radius depending on how many pods there are, if so I can also then fix up the building size so it's appropriate.

Probably for demos it'd make sense to turn off the 'Destroy Pods' option in the preferences.

And you should practice more.  :-)

**Comment by davsclaus** [comment]
Well you wanna sometimes show the HA / resillent when pods are killed, and doing that by playing angry pods would be cool. Just that you dont want to kill those "core" pods.

**Comment by gashcrumb** [comment]
Yeah, good point!  Will figure out a blacklist...

**Comment by gashcrumb** [comment]
K, made the pods spawn a bit closer but hopefully not right on top of you, depends on how many pods are running.  Also added a blacklist setting with some defaults, it's in the preferences.  Haven't dealt with the world generation or added player health yet, but if you're feeling daring you can pull the latest fabric8-console docker image.

**Comment by davsclaus** [comment]
Thanks @gashcrumb I was a chicken and used the current version of the console. But great to hear of the new stuff already in it.

**Comment by gashcrumb** [comment]
@davsclaus no worries, probably better you didn't, had some other issues come up after pulling in the newer patternfly in the console project :-)
